world.js

```diff
@@ -92,9 +90,6 @@ world.connectRoom = async u => {
       wsrtc.removeEventListener('init', init);
       
       localPlayer.bindState(state.getArray(playersMapName));
-      if (mediaStream) {
```

Comment:
Does this lose mic functionality?

Reply:
No, and it was preventing multiplayer from working at all there, the code is in character-controller.js now
![image](https://user-images.githubusercontent.com/18633264/162920479-8dbb2c44-fb33-4d29-aa37-ea7039bd8987.png)